source/languages/en/riakcs/cookbooks/Usage-and-Billing-Data.md

```diff
@@ -209,25 +209,44 @@ script.
 
 #### Regular Schedules
 
-If you would like to have an Riak CS node calculate the storage used by
-every user at the same time (or times) each day, specify a schedule in
-that node's Riak CS `app.config` file.
+If you would like to have an Riak CS node calculate the storage used by every
+user at the same time (or times) each day, specify a schedule in that node's
+Riak CS `riak-cs.conf` file, or in the old-style `advanced.config` or
+`app.config` file.
 
 In the `riak_cs` section of the file, add an entry for
 `storage_schedule` like this:
 
-```erlang
+```riakcsconf
+stats.storage.schedule.1 = "06:00"
+```
+
+```advancedconfig
+{storage_schedule, "0600"}
+```
+
+```appconfig
 {storage_schedule, "0600"}
 ```
 
-The time is given as a string of the form `HHMM`, representing the hour
-and minute GMT to start the calculation process. In this example, the
+The time is given as a string of the form `HH:MM` (omit the `:` in the old-style
+`advanced.config`/`app.config` files, so taht it's just `HHMM`), representing
+the hour and minute GMT to start the calculation process. In this example, the
 node would start the storage calculation at 6am GMT every day.
 
-To set up multiple times, specify a list in the schedule. For example,
+To set up multiple times, simply specify multiple times. For example,
```

Comment:
That may be valuable, but I'm not familiar with the performance impact of these operations. There wasn't any note there in the original docs, so I'm going to cop-out and say that adding such a note isn't within the scope of this issue.
I would love to see a new issue opened up so I remember to look into this later.